docs/source/Plugins/fun.rst

```diff
@@ -36,6 +36,11 @@ Magic 8 Ball
 .. automodule:: plugins.downloaded.magic_ball.main
    :members:
 
+Basketball
+^^^^^
```

Comment:
Add the `^^^` to underline the title completely

Reply:
I just changed it. I think now it's ok